### ISSUE TITLE
How to warm up the segmentation model upper-net?

### ISSUE THREAD

**Issue by DZhaoXd** [boschresearch/ALDM#6]
Great work! I would like to train your model on my own data, but need a warm-up model for the segmentation model upper-net. Can you provide the code for training the segmentation model during the warm-up stage?  Thanks

**Comment by YumengLi007** [comment]
Hi @DZhaoXd , thanks for your interests in ALDM! 
For the segmenter, we used the ADE20K pretrained model from [semantic-segmentation-pytorch](https://github.com/CSAILVision/semantic-segmentation-pytorch). 
Similar to the weights modification in ControlNet [tool_add_control.py](https://github.com/lllyasviel/ControlNet/blob/main/tool_add_control.py), you could then add one more channel in the final classification head of your trained segementer for the extra "fake" class

**Comment by DZhaoXd** [comment]
Thank you for your positive response. I will try it.
